src/features/offer/api/useSimilarOffers.native.test.ts

```diff
@@ -181,15 +183,17 @@ describe('useSimilarOffers', () => {
         expect(getRelatedProductsSpy).not.toHaveBeenCalled()
       })
 
-      it('when offer and category excluded provided', () => {
+      it('when offer and category excluded provided', async () => {
         renderHook(() =>
           useSimilarOffers({
             offerId: mockOfferId,
             categoryExcluded: SearchGroupNameEnumv2.FILMS_SERIES_CINEMA,
             shouldUseAlgoliaRecommend: true,
           })
         )
-        expect(getRelatedProductsSpy).not.toHaveBeenCalled()
+        await waitFor(() => {
+          expect(getRelatedProductsSpy).not.toHaveBeenCalled()
+        })
```

Comment:
Vu que c'est un `not.toHaveBeenCalled()`, je préfère utiliser `act` plutôt que `waitFor` : 
```
        await act(async () => {})
        expect(getRelatedProductsSpy).not.toHaveBeenCalled()
```